docs/usage/config-overview.md

```diff
@@ -0,0 +1,209 @@
+# Renovate Configuration Overview
+
+When Renovate runs on a repository, the final config in use is derived from:
+
+- Global config
+- Inherited config
+- Repository config
+- Resolved presets referenced in config
+
+## Types of Config
+
+### Global config
```

Comment:
I think we should a reference that it is sometimes called  bot config. 

So people know that they can search the term in our discussions.

Reply:
Good point. I was even considering starting with a glossary of terms at the top of this page

Reply:
If we can standardize on the terms, and use those consistently, we can add those terms to the [`abbreviations.md` file in the Renovate docs repository](https://github.com/renovatebot/renovatebot.github.io/blob/main/includes/abbreviations.md). That way when you read the docs online, you'll see a explanation when you hover over the term.

Also see the [Material for MkDocs manual, adding abbreviations and a glossary](https://squidfunk.github.io/mkdocs-material/reference/tooltips/#adding-abbreviations).

Or we can do both:

- Have glossary in this file directly
- Put the standardized terms in the `abbreviations.md` file

Reply:
I tried to use capitalization for "Global config", "Inherited config", "Onboarding config", etc. It would be good if these had hover explanations when viewed in docs

Reply:
Added bot config clarification 2 lines down